packages/slice-machine/lib/models/common/widgets/Group/ListItem/index.js

```diff
@@ -24,6 +24,7 @@ import Hint from "@lib/builders/common/Zone/Card/components/Hints";
 import ListItem from "@components/ListItem";
 import { createFriendlyFieldNameWithId } from "@src/utils/fieldNameCreator";
 
+/* eslint-disable */
```

Comment:
disable for the whole file?

Reply:
Every line of this file has an ignore linter comment :p Instead of stacking many ignore comments I prefer to disable the linter on the whole file. We will remove all the comments when migrating this file to TS, for now it doesn't have so much value to maintain it :) 

You agree ?

Reply:
yeah